examples/python/xds/README.md

```diff
@@ -121,3 +121,20 @@ Add the `--secure true` flag to the invocation outlined above.
 ```
 python client.py xds:///my-backend --secure true
 ```
+
+## Building Docker
+
+From the gRPC workspace folder:
+
+Client:
+```
+docker build -f examples/python/xds/Dockerfile.client -t "us-docker.pkg.dev/grpc-testing/examples/xds-example-python-client" .
+```
+
+Server:
+```
+docker build -f examples/python/xds/Dockerfile.server -t "us-docker.pkg.dev/grpc-testing/examples/xds-example-python-server" .
```

Comment:
This documentation is end-user facing and, in general, end users will not have access to the `grpc-testing` project. Maybe we can use a placeholder for the tag in this documentation?

Reply:
Done.